docs/admin/logical-replication.rst

```diff
@@ -109,13 +110,14 @@ is not supported.
 Security
 --------
 
-To create a publication, a user must have the ``AL`` privilege on the cluster.
-To add tables to a publication, the user must have ``DQL``, ``DML``, ``DDL``
-privileges on the table. When user creates a publication that publishes all
-tables automatically, only those tables where user has ``DQL``, ``DML``,
-``DDL`` privileges will be published.
+To create, alter or drop a publication, a user must have the ``AL`` privilege
+on the cluster. To add tables to a publication, the user must have
+``DQL``, ``DML``, ``DDL`` privileges on the table. When user creates a
+publication that publishes all tables automatically, only those tables where
+user has ``DQL``, ``DML``, ``DDL`` privileges will be published.
```

Comment:
Think we should add:
Only the owner (the user who created the publication) or a superuser is allowed to `ALTER` or `DROP` a publication.

And same for the subscription section below.